gitea-formula/pillar.example

```diff
@@ -0,0 +1,10 @@
+# The "gitea" pillar allows for all Gitea configuration options:
+# https://docs.gitea.com/administration/config-cheat-sheet
+#
+# Some defaults will be written (unless specified otherwise) to accomodate the default configuration shipped with the package.
+
+gitea:
+  default:
+    app_name: GitCoffee
+  server:
+    http_addr: 127.0.0.1
```

Comment:
Would it make sense to put the config under a sub-key like `config:`? (Without that, it's basically impossible to introduce any pillar that should not end up in the config file.)

Reply:
Will there be any need for such non-config pillar data?

Reply:
Well, I'd say the question is "are you absolutely sure there will never be a need for non-config pillar data?"

Just as an example: Maybe one day you want to introduce an option to disable getea.service?

Reply:
Ok.